apis/proto/v1/manager/replication/agent/replication_manager.proto

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright (C) 2019-2020 Vdaas.org Vald team ( kpango, rinx, kmrmt )
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package replication_manager;
+
+option go_package = "github.com/vdaas/vald/apis/grpc/v1/manager/replication/agent";
+option java_multiple_files = true;
+option java_package = "org.vdaas.vald.api.v1.manager.replication.agent";
+option java_outer_classname = "ValdReplicationManagerAgent";
+
+import "apis/proto/v1/payload/payload.proto";
+import "google/api/annotations.proto";
+
+
+service Replication {
+
+  rpc Recover(payload.Replication.Recovery) returns (payload.Empty)  {
+    option (google.api.http).post = "/replication/recover";
```

Comment:
I'm sorry if I made a mistake.
Do you need a post body?

Reply:
replicatiom component proto is just copy of v0 api please ignore it